### PR TITLE
Migrate S2189, S2681, S3782, S3796, S1528, S1529, S3524 to messageId

### PR DESCRIPTION
**Pull request by yassin-kammoun-sonarsource**

### CODE REVIEW

File: eslint-bridge/src/rules/bitwise-operators.ts [hunk 1]
```diff
@@ -42,6 +42,12 @@ const BITWISE_OPERATORS = [
 ];
 
 export const rule: Rule.RuleModule = {
+  meta: {
+    messages: {
+      reviewBitwise:
+        'Review this use of bitwise "{operator}" operator; conditional "{{intended}}" might have been intended.',
```
Comment (vilchik-elena) [inline eslint-bridge/src/rules/bitwise-operators.ts:48]: why only one set of curly braces?

File: eslint-bridge/src/rules/no-unenclosed-multiline-block.ts [hunk 2]
```diff
@@ -140,9 +155,11 @@ function raiseInlineAndIndentedIssue(
   context: Rule.RuleContext,
 ) {
   context.report({
-    message:
-      `This line will not be executed ${chainedStatements.includedStatementQualifier()}; only the first statement will be. ` +
-      `The rest will execute ${chainedStatements.excludedStatementsQualifier()}.`,
+    messageId: 'onlyFirstStatement',
```
Comment (vilchik-elena) [inline eslint-bridge/src/rules/no-unenclosed-multiline-block.ts:158]: should be `onlyFirstLine`?

**Review by vilchik-elena — CHANGES_REQUESTED** [review]

File: eslint-bridge/src/rules/no-unenclosed-multiline-block.ts (hunk 2 above)
Comment (yassin-kammoun-sonarsource) [reply eslint-bridge/src/rules/no-unenclosed-multiline-block.ts:158]: I don't think so. It raises issues on the following pattern:

```javascript

if (condition) action1();
  action2();
```

**Comment by vilchik-elena** [comment]
Closing PR as we realized that we don't have to migrate to message IDs

**Pull request closed without merging**